### ISSUE TITLE
Semulov is now in maintenance mode

### ISSUE THREAD

**Issue by kainjow** [kainjow/Semulov#47]
Semulov was written when Macs shipped with DVD drives, iPods were mountable on your desktop and FireWire was everywhere. But, times have changed. Personally I don't use any ejectable media anymore, so this app is really no longer useful to me. Therefore, I don't plan on adding any more features to it. I will try to keep bugs fixed though. For now I will close out feature requests as `wontfix`. If anyone's interesting in adding functionality please file a pull request and I will consider them.

**Comment by sn0wyfall** [comment]
Plenty of people use removable media nowadays in the form of external drives. I often have about 10 connected to my mac at any time.

**Comment by erikw** [comment]
Tip; there's also https://www.stclairsoft.com/Jettison/ which can eject attached disks!

**Comment by kmbst** [comment]
Just popped by to say that your software is truly appreciated. I use a USB-C external monitor with an external SSD connected to it. Whenever I need to unplug my macbook (which is multiple times per day) I use your app to one-click eject, rather than opening a Finder window and clicking on the drive to eject it. Plus it reminds me not to disconnect my Mac without ejecting properly first thanks to the little number next to the icon, which is a life saver.

Truly thankful for your app. You have some fans out there.

**Comment by kavehv** [comment]
Yes, I just wanted to drop by and also express my thanks for this cool little utility.  I'm sure, given the MIT license, someone will fork and maintain it (maybe even me if I have spare time).  Cheers!

**Comment by RetroWindow** [comment]
There is an issue when trying to open v2.6 on a 2018 Intel Mac Mini running Big Sur 11.6.4.  A pop-up window appears stating that Apple "cannot check it for malicious software.   This software needs to be updated."  v2.4 opens and runs successfully.

I don't know whether this issue qualifies as "maintenance."  But I do want to let you know that I appreciate your work whether or not this issue can be addressed.

**Comment by kainjow** [comment]
@RetroWindow I never setup Semulov with notarization, which is required now for Mac apps outside of the App Store, otherwise you'll see that dialog you mentioned. You can get around it by right-clicking the app in Finder and selecting the Open context menu.

**Comment by RetroWindow** [comment]
Hi Kevin-

Success!  Thank you.

I forgot to mention that after right-clicking the Semulov icon, I had to also right-click Open to access the option for opening a non-notarized app.

Best regards-

RW
